aws-lambda-layer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM lambci/lambda:build-ruby2.5
+
+ARG FREETDS_VERSION=1.1.6
+
+ENV INSTALLDIR='/tmp/freetds'
+ENV FREETDS_VERSION=$FREETDS_VERSION
+
+RUN yum update -y
+RUN yum install wget build-essential libc6-dev -y
+
+RUN mkdir $INSTALLDIR
+
+RUN wget http://www.freetds.org/files/stable/freetds-${FREETDS_VERSION}.tar.gz && \
```

Comment:
probably want to do this: https://github.com/veracross/freetds/blob/master/Dockerfile#L6